### PR TITLE
Fix typos in var/

### PR DESCRIPTION
**Pull request by szepeviktor**
This PR fixes misspellings in var/

@douglasjacobsen

### CODE REVIEW

File: .github/workflows/spelling.yaml [hunk 1]
```diff
@@ -15,5 +15,5 @@ jobs:
     - name: Check Spelling
       uses: crate-ci/typos@7ad296c72fa8265059cc03d1eda562fbdfcd6df2 # v1.9.0
```
Comment (szepeviktor) [inline .github/workflows/spelling.yaml:16]: v1.9.0 is a very old release, maybe we need an upgrade.
https://github.com/crate-ci/typos/releases

**Review by douglasjacobsen — APPROVED** [review]

**Comment by douglasjacobsen** [comment]
@szepeviktor Thanks for these contributions!

**Comment by szepeviktor** [comment]
Glad to contribute.

**Pull request merged**